haproxy_stats.sh

```diff
@@ -1,25 +1,16 @@
 #!/bin/bash
 set -o pipefail
 
-if [[ "$1" = /* ]]
-then
-  HAPROXY_SOCKET="$1"
-  shift 1
-fi
+[ "$#" -gt 3 ] && HAPROXY_SOCK="$(echo $1 | tr -d '\040\011\012\015')" && shift 1
+
+# Define variables and perform checks
+CONFIG_FILE="$(dirname $(readlink -f "$0"))/haproxy_config"
+. $CONFIG_FILE
```

Comment:
yeah, ok, number of arguments check is sufficient..
I do like a bit more validation though, similarly to `haproxy_discovery.sh`
```
[ "$#" -gt 3 ] && echo $1 | grep -q ^/ && HAPROXY_SOCKEt="$(echo $1 | tr -d '\040\011\012\015')" && shift 1
```